docs/packages/only_hunter/AllTheFlopsThreads.rst

```diff
@@ -0,0 +1,10 @@
+.. spelling::
+
+    AllTheFlopsThreads
+
+.. _pkg.AllTheFlopsThreads:
+
+AllTheFlopsThreads
+==================
+
+TODO
```

Comment:
Template can be helpful here, like:

```
-  `__FIXME__ Official <https://__FIXME__>`__
-  `__FIXME__ Hunterized <https://github.com/hunter-packages/__FIXME__>`__
-  `__FIXME__ Example <https://github.com/ruslo/hunter/blob/master/examples/__FIXME__/CMakeLists.txt>`__
-  Available since `__FIXME__ vX.Y.Z <https://github.com/ruslo/hunter/releases/tag/vX.Y.Z>`__
-  Added by `__FIXME__ <https://github.com/__FIXME__>`__ (`__FIXME__ pr-N <https://github.com/ruslo/hunter/pull/N>`__)

.. code-block:: cmake

    hunter_add_package(__FIXME__)
    find_package(__FIXME__ CONFIG REQUIRED)
    target_link_libraries(foo __FIXME__::__FIXME__)
```

Reply:
> Available since `__FIXME__ vX.Y.Z <https://github.com/ruslo/hunter/releases/tag/vX.Y.Z>`__

We can use badge here:
* https://github.com/ruslo/hunter/issues/513